README.md

```diff
@@ -90,109 +92,78 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
-
-> Describe between 1-3 key features of the application.
+<!-- LIVE DEMO -->
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+## 🚀 Live Demo <a name="live-demo"></a>
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+- [Live Demo Link](https://moneycast.onrender.com)
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+## 🚀Video <a name="live-demo"></a>
 
-## 🚀 Live Demo <a name="live-demo"></a>
+- [Video presentation](https://www.loom.com/share/5e11347d895d4630afac0625bd35bb9b)
 
```

Comment:
Hello @uwadonat,
thanks for being professional 
i do know  that I missed that part of the video but It was because for some reason loom dose not record my vs code i tried in previous videos and i keep recording chrome, even after switching to vs code 

I'm having an issue with zoom it doesn't share my vs code tab